tests/api/test_api.py

```diff
@@ -1,70 +1,263 @@
+import json
+
+import pytest
+import requests
+
 import cript
+from cript.api.exceptions import InvalidVocabulary, InvalidVocabularyCategory
+from cript.api.paginator import Paginator
+from cript.nodes.exceptions import CRIPTNodeSchemaError
+
+
+def test_create_api() -> None:
+    """
+    tests that an API object can be successfully created with host and token
+    """
+    api = cript.API("http://development.api.mycriptapp.org/", "123456789")
```

Comment:
I refined it to use env variables

Reply:
I think I like the idea of a config.py instead, I think that is going to be the easiest option possible down the line